screenpipe-vision/src/core.rs

```diff
@@ -156,47 +162,39 @@ pub async fn continuous_capture(
         previous_image = Some(Arc::new(image.clone()));
         // debug!("ocr_task_running {} BEFORE if if !ocr_task_running.load(Ordering::SeqCst)", ocr_task_running.load(Ordering::SeqCst));
 
-        if !ocr_task_running.load(Ordering::SeqCst) {
```

Comment:
fyi this atomic bool stuff was not necessary

it was used back in the day we did ocr on multi thread but here its just sequential, no need